handlers/africastalking/africastalking.go

```diff
@@ -56,14 +56,14 @@ func (h *handler) receiveMessage(ctx context.Context, channel courier.Channel, w
 	form := &moForm{}
 	err := handlers.DecodeAndValidateForm(form, r)
 	if err != nil {
-		return nil, err
+		return nil, courier.WriteAndLogRequestError(ctx, w, r, channel, err)
```

Comment:
properly return 400 error response 

this fix https://sentry.io/nyaruka/textit-courier/issues/418551945/